src/main/resources/templates/db_scripts/schema.json

```diff
@@ -1,6 +1,11 @@
 {
   "exactCount" : 20000,
   "scripts": [
+    {
+      "run": "before",
+      "snippetPath": "remove_from_rmb_internal_index.sql",
```

Comment:
This is a workaround to be able to create default indexes by foreign keys instead of  removed indexes for jsonb fields: fiscalYearId, rolloverLedgerId in ledger_fiscal_year_rollover_budget. Without this removal from rmb_internal_index the indexes are not created.